structure/heap/heap_test.go

```diff
@@ -0,0 +1,100 @@
+package heap
+
+import (
+	"testing"
+)
+
+type _int int
```

Comment:
Why? This convention is very different from Go standards, please don't use `underscore` to mark things private 🙏🏼 

```suggestion
type testInt int
```